apps/src/dom.js

```diff
@@ -37,14 +37,14 @@ exports.getTouchEventName = function(eventName) {
 };
 
 var addEvent = function(element, eventName, handler) {
-  element.addEventListener(eventName, handler, false);
-
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
```

Comment:
The challenge is that touch and mouse aren't mutually exclusive.  E.g. the Chromebook Pixel supports touch events on its touchscreen but also has a trackpad.  If we don't also bind mouse events we'll break mouse interaction for these devices.